resources/keywords/create_provider_account.resource

```diff
@@ -51,8 +53,11 @@ User Enters Provider Account Name And Selects Database Provider
     ${dbprovider}=    Set Variable    ${isv}
     Wait Until Page Contains Element    ${drpDwnDBProvider}    timeout=20s
     ${paname}=    Get Provider Account Name    ${dbprovider}    ${invalid}
-    Set Test Variable    ${provaccname}    ${paname}
-    Input Text    ${txtBxName}    ${provaccname}
+    Run Keyword If    ${invalid}
```

Comment:
I hope we can avoid this if we combine the keywords on create_instance.resource file